.github/workflows/examples-efr32.yaml

```diff
@@ -81,7 +81,7 @@ jobs:
                 --enable-flashbundle \
                 --target efr32-brd4187c-light-use_ot_lib \
                 --target efr32-brd4187c-pump \
-                --target efr32-brd4187c-lock-rpc-shell-enable_heap_monitoring \
```

Comment:
As a curiosity, was rpc the wrong acronyme to begin with or was the lock really behaving as a radio co-processor in this build configuration?